deps/config/types.go

```diff
@@ -500,3 +521,41 @@ type ApisConfig struct {
 	// Chain API auth secret for the Curio nodes to use.
 	StorageRPCSecret string
 }
+
+type CurioBatchingConfig struct {
+	// Precommit Batching configuration
+	PreCommit PreCommitBatchingConfig
+
+	// Commit batching configuration
+	Commit CommitBatchingConfig
+}
+
+type PreCommitBatchingConfig struct {
+	// Enable / Disable Precommit aggregation
+	AggregatePreCommits bool
+
+	// Maximum precommit batch size - batches will be sent immediately above this size if BaseFeeThreshold is higher
+	// than the current base fee. If not then we will wait batch if forced due to PreCommitBatchSlack
+	MaxPreCommitBatch int
+
+	// Time buffer for forceful batch submission before sectors/deal in batch would start expiring
+	PreCommitBatchSlack Duration
+
+	// Base fee value below which we should try to send Precommit message. This will be ignored if PreCommitBatchSlack has reached
+	BaseFeeThreshold types.FIL
+}
+
+type CommitBatchingConfig struct {
+	// Enable / Disable commit aggregation
+	AggregateCommits bool
+
+	// Maximum batched commit size - batches will be sent immediately above this size if BaseFeeThreshold is higher
+	//	// than the current base fee. If not then we will wait batch if forced due to CommitBatchSlack
+	MaxCommitBatch int
+
+	// Time buffer for forceful batch submission before sectors/deals in batch would start expiring
+	CommitBatchSlack Duration
+
+	// Base fee value below which we should try to send Commit message. This will be ignored if CommitBatchSlack has reached
+	BaseFeeThreshold types.FIL
+}
```

Comment:
Let's do that differently in Curio:
* Don't give this choice of enable/disable
* Automatically detect the better thing to do
  * Simulate both sends, detect which one is more efficient in Fees

Also reorder/cleanup names a bit.

```suggestion
type PreCommitBatchingConfig struct {
	// Base fee value below which we should try to send Precommit messages immediately
	BaseFeeThreshold types.FIL
	
	// Maximum amount of time any given sector in the batch can wait for the batch to accumulate
	Timeout Duration
	
	// Time buffer for forceful batch submission before sectors/deal in batch would start expiring
	Slack Duration
}

type CommitBatchingConfig struct {
	// Base fee value below which we should try to send Commit messages immediately
	BaseFeeThreshold types.FIL
	
	// Maximum amount of time any given sector in the batch can wait for the batch to accumulate
	Timeout Duration
	
	// Time buffer for forceful batch submission before sectors/deals in batch would start expiring
	Slack Duration
}
```

Reply:
Fixed